workshop/Lab0/README.md

```diff
@@ -69,6 +69,41 @@ $echo $PATH
 
 3. Convert the binary file to an executable: `chmod +x /usr/local/bin/kubectl`
 
+# Configure Kubectl to point to IBM Cloud Kubernetes Service
+1. List the clusters in your account:
+
+```console
+ibmcloud ks clusters
+```
+
+2. (optional) Set an environment variable for convenience
```

Comment:
It seems that there are additional places beyond the next command where `$CLUSTER_NAME` is used so I would remove the "optional"